packages/rollup/index.ts

```diff
@@ -1,4 +1,4 @@
 export * from './src/generators/init/init';
 export * from './src/generators/rollup-project/rollup-project';
-export * from './src/executors/rollup/schema';
+export type * from './src/executors/rollup/schema';
```

Comment:
Same here export `RollupExecutorOptions` `Globals` and `AssetGlobPattern`

Reply:
Also done!